doc/rtd/reference/faq.rst

```diff
@@ -15,21 +15,15 @@ Having trouble? We would like to help!
 - Find a bug? Check out the :ref:`reporting_bugs` topic to find out how to
   report one
 
-``autoinstall``, ``preruncmd``, ``postruncmd``
-==============================================
-
-Since ``cloud-init`` ignores top level user data ``cloud-config`` keys, other
-projects such as `Juju`_ and `Subiquity autoinstaller`_ use a YAML-formatted
```

Comment:
As we are removing all the references to Juju in this page, could we also remove the references to them, please?

* https://github.com/canonical/cloud-init/blob/8ceae8b7b504960e348ecda36c26db0b6fa12890/doc/rtd/reference/faq.rst?plain=1#L86
* https://github.com/canonical/cloud-init/blob/8ceae8b7b504960e348ecda36c26db0b6fa12890/doc/rtd/reference/faq.rst?plain=1#L109